app/api/helpers/calendar/ical.py

```diff
@@ -8,7 +7,7 @@
 from app.models.session import Session
 
 
-def to_ical(event, include_sessions=False, my_schedule=False):
+def to_ical(event, include_sessions=False, my_schedule=False, user_id=current_user.id):
```

Comment:
Default should be none. Python default arguments are evaluated once on load time, so this will never work

Reply:
Yes just pushing that change only :sweat_smile: realised that after pushing :p